core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1170,6 +1170,10 @@ class KafkaApis(val requestChannel: RequestChannel,
           debug(s"OffsetRequest with correlation id $correlationId from client $clientId on partition $topicPartition " +
               s"failed because the partition is duplicated in the request.")
           buildErrorResponse(Errors.INVALID_REQUEST, partition)
+        } else if (partition.timestamp() == ListOffsetsRequest.EARLIEST_LOCAL_TIMESTAMP && version < 8) {
+          throw new UnsupportedVersionException(s"apiVersion must be >=8 for EARLIEST_LOCAL_TIMESTAMP")
```

Comment:
Could you please build error response to the partition which has unsupported content? throwing exception will break whole request

Reply:
Thank you. Update it.